src/components/editions/headline.tsx

```diff
@@ -52,8 +52,38 @@ const standardFontStyles = css`
 		${headline.medium({ lineHeight: 'tight' })}
 	}
 `;
+const interviewStyles = css`
+	margin-left: 0.75rem;
```

Comment:
We should probably use a `remSpace` value here

Reply:
Nice spot  - converted.